.github/scripts/create_and_push_docker_image.sh

```diff
@@ -15,50 +15,28 @@
 # limitations under the License.
 #
 
-INPUT_IMAGE_NAME=$1
-INPUT_IMAGE_TAG=$2
-INPUT_REPO_DIR=$3
-INPUT_NOTEBOOK_USER=$4
+INPUT_DOCKER_FULL_IMAGE_NAME=$1
+INPUT_PUSH=$2
 
-# Check for docker image name
-if [ -z "$INPUT_IMAGE_NAME" ]; then
-  echo "Input the image name for the docker image as the first parameter"
-  exit 1
-fi
+# ghcr.io/projectnessie/nessie-binder-demos:e12478cfb3447af4bc26e535a9daffe374233aa1
+DOCKER_IMAGE_NAME=$(echo "${INPUT_DOCKER_FULL_IMAGE_NAME}" | cut -f1 -d":")
```

Comment:
Can we add a check here if the input docker image name has been passed?

Reply:
ok i will add it back